frontend/src/components/Outdated.tsx

```diff
@@ -3,6 +3,7 @@ import type { WebxdcStatusUpdatePayload } from '../bindings/WebxdcStatusUpdatePa
 
 interface OutdatedViewProps {
   updated_received: boolean
+  serial: number
```

Comment:
The bot will now only send the newest updates to updated instances instead of the whole AppIndex.